app/models/miq_server.rb

```diff
@@ -627,10 +627,12 @@ def messaging_health_check
     broker = MiqQueue.messaging_client("health_check")
     return if broker.nil?
 
-    broker.publish_message(:service => "manageiq.liveness-check", :message => "test message", :payload => {})
-    broker.subscribe_messages(:service => "manageiq.liveness-check") { break }
+    # Fail health check if list of topics can't be retrieved
+    broker.topics
```

Comment:
@agrare Should we log on success here? The push/pop approach had logging baked in by the messaging client

Reply:
I think we're okay to just log on failure